CMakePresets.json

```diff
@@ -2,19 +2,148 @@
   "version": 3,
   "configurePresets": [
     {
-      "name": "Ninja",
-      "generator": "Ninja Multi-Config",
+      "name": "default",
+      "hidden": true,
+      "generator": "Ninja",
+      "binaryDir": "out/build/${presetName}",
+      "cacheVariables": {
+        "Languages_Description": "Modify the setting to any language you want: CHINESE;DUTCH;ENGLISH;FRENCH;GERMAN;ITALIAN;POLISH;RUSSIAN",
+        "Languages": "ENGLISH",
+        "CMAKE_RUNTIME_OUTPUT_DIRECTORY_Description": "Point the setting to your game directory, so you can run/debug the executables directly from there",
+        "CMAKE_RUNTIME_OUTPUT_DIRECTORY": "."
+      },
       "architecture": {
         "value": "x86",
         "strategy": "external"
       }
-    }
-  ],
-  "buildPresets": [
+    },
+    {
+      "name": "JA2",
+      "hidden": true,
+      "inherits": "default",
+      "cacheVariables": {
+        "Applications": "JA2"
+      }
+    },
+    {
+      "name": "JA2-Debug",
+      "displayName": "JA2 Debug",
```

Comment:
Also had to introduce displayName as spaces and dashes are differently sorted by VS and with dashes it would have issues, while spaces work